src/views/header/HeaderBlock.js

```diff
@@ -30,7 +30,11 @@ const View = boneView.extend({
   render: function() {
     this.renderSubviews();
 
-    return this.el.className = "biojs_msa_header";
+    this.el.style.height = (
```

Comment:
This dynamically adjusts the height of the top-level header block based on pinned annotations.